### DISCUSSION TITLE
How to use wlan1 instead of wlan0

### DISCUSSION THREAD

**Discussion by nikallian** [RaspAP/raspap-webgui#795] [Q&A]
Hi

I installed raspap on a existing installation of raspbian on a raspberry pi 3B+ which I was using for various other services so I didn't want to ruin my setup. So I tried by editing the config files of raspap to make it work using a USB WiFi as wlan1. Despite my efforts I couldn't have it up and running. Actually, the GUI shows mixed options with wlan0 and wlan1. Moreover, while AP shows up and running and I am able to connect to it, there is IPv4 assignment. 

Is there any guide on how to setup raspap with wlan1 having eth0 (in my case it's bond0 since I have bonded wlan0/eth0 interfaces, that's the reason I can't use the internal wifi interface of my raspberry) as upstream (internet) interface ?

Thanks in advance for your help

**Comment by billz** [comment]
There is a FAQ that addresses this specifically https://docs.raspap.com/faq/#interfaces
Also discussed here https://docs.raspap.com/defaults/#networking-defaults

**Reply by nikallian** [reply]
Oh! Great!

Thanks for the prompt reply!

Kind Regards

**Comment by nikallian** [comment]
Hello Again,

I finally managed to have it up and running without making big changes to my rpi networking configuration. There was a lot of things needed to be changed except those mentioned in the documentation in order to work fully customized. 

Currently I have one issue not so important but I would like to find the cause. Below you can see that Hotspot menu doesn't give me any option (buttons) to save my configuration. 

![image](https://user-images.githubusercontent.com/78311216/106446650-2b9def80-6489-11eb-960b-e4fb140c289c.png)

Any ideas on why this might be happening ?

thanks in advance!

**Reply by nikallian** [reply]
Just an update, 

I don't see buttons (Save, restart) in many other tabs as the DHCP, AdBlocking e.t.c. I followed the full manual installation in order to be sure that no changes to my system will be done without my knowledge and approval. 

Thank you

**Reply by billz** [reply]
RaspAP is configured in monitor mode. This is all covered in the docs (try the search function)
https://docs.raspap.com/faq/#monitor
https://docs.raspap.com/defaults/#managing-config-values

**Reply by nikallian** [reply]
I am so sorry I just saw it my self. I was my mistake during the custom configuration. I just corrected it. 

Thank you once more!

**Reply by billz** [reply]
No problem 😉 There are a lot of configuration options with a custom install.
